src/views/AddItem.jsx

```diff
@@ -1,3 +1,87 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	// sets initial default values in form fields and deconstruct form field variables
+	const initialState = { itemName: '', estimate: '7' };
+	const [formData, setFormData] = useState(initialState);
+	const { itemName, estimate } = formData;
+
+	// for now, we retrieve the token for the test list from local storage;
+	const token = window.localStorage.getItem('tcl-shopping-list-token');
+
+	// updates state based on user input for multiple form fields
+	const formHandler = (e) => {
+		const name = e.target.name;
+		const value = e.target.value;
+		setFormData((formData) => ({ ...formData, [name]: value }));
+	};
+
+	// if no item name provided, gives user feedback to add item
+	// if item name provided, adds item to user's list and gives user success feedback
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		if (itemName === '') {
+			alert('Please enter the name of your item.');
+		} else {
+			// changes string to number, as required by addItem function
+			const daysUntilNextPurchase = +estimate;
+			// uses addItem function imported from api; this takes 2 arguments: the user's token and the item object containing item name and estimate of next purchase date
+			addItem(token, { itemName, daysUntilNextPurchase });
+			alert(`You've added ${itemName} to your shopping list!`);
+			//Clear Form Data
+			setFormData(initialState);
+		}
+	};
+
+	// displays a form with a text field for item name and 3 radio buttons for user to choose next purchase date estimate
+	return (
+		<div>
+			<h1>Add a New Item</h1>
+			<form onSubmit={handleSubmit}>
+				<label htmlFor="itemName">Name:</label>
+				<input
+					type="text"
+					name="itemName"
+					id="itemName"
+					value={itemName}
+					onChange={formHandler}
+				/>
+
+				<fieldset>
+					<legend>How soon will you buy this again?</legend>
+					<label htmlFor="estimate">
+						<input
+							type="radio"
+							value="7"
+							id="soon"
+							name="estimate"
+							onChange={formHandler}
+							checked={estimate === '7'}
+						/>
+						<label htmlFor="soon">Soon</label>
+						<input
+							type="radio"
+							value="14"
+							id="kindOfSoon"
+							name="estimate"
+							onChange={formHandler}
+							checked={estimate === '14'}
+						/>
+						<label htmlFor="kindOfSoon">Kind of Soon</label>
+						<input
+							type="radio"
+							value="30"
+							id="notSoon"
+							name="estimate"
+							onChange={formHandler}
+							checked={estimate === '30'}
+						/>
+						<label htmlFor="notSoon">Not Soon</label>
+					</label>
```

Comment:
# Problem

I'd like to call your attention to the concept of [ 🎱 `magic numbers`](https://en.wikipedia.org/wiki/Magic_number_(programming).)

Right now there are a lot of great context clues that define the relationship of the `estimate` string and number value because of the way the `form inputs` are set up. But let's take a step back and think about the future use case of retrieving this number from the database, and using it to do downstream manipulations. All of a sudden we'll have a "contextless number". A new engineer extending your work would have to read through the past issues and venture into the `AddItem` component to understand the context.

# How to solve the problem
To lighten the mental load for other developers, you can communicate this explicit relationship context through code. You could do this by:
1. creating some sort of constant object that clearly have keys with the "soon, kinda soon, not soon" with number values
1. storing constant values within a `consts.js` file 
1. etc.

There's lots of ways you can do this, so this is where the team discussion will come in. Feel free to reach out if y'all need some mentor inspiration for this ask. 

# Why should you solve the problem

Communicating relationships and context through code is high up in priority (like good naming) when thinking through how to be a ✨ team focused code communicator ✨. Also, this will help yourself when 6 months later you forget a lot of crucial context about code YOU wrote yourself. (Ask my how I know this 😅)

Reply:
I'm hoping that we understood and implemented correctly what you're referring to here with our recent commit; if not I will reach out for help.